ui/extractedTranslations/it/translation.json

```diff
@@ -0,0 +1,252 @@
+        "hour": "Ore",
+        "minutes": "Minuti",
+        "months": "Mesi",
+        "seconds": "Secondi"
+      },
+      "title": "Scansioni periodiche"
+    },
+    "scanner": {
+      "description": "Verranno effettuate delle scansioni per tutti gli utenti alla ricerca di nuovi o aggiornati media",
+      "scan_all_users": "Scansiona per tutti",
+      "title": "Scansioni"
+    },
+    "user_preferences": {
+      "change_language": {
+        "description": "Cambia la lingua di Photoview per questo utente",
+        "label": "Lingua Photoview"
+      },
+      "language_selector": {
+        "placeholder": "Seleziona lingua"
+      },
+      "title": "Preferenze utente"
+    },
+    "users": {
+      "add_user": {
+        "submit": "Aggiungi utente"
+      },
+      "confirm_delete_user": {
+        "action": "Elimina {{user}}",
+        "description": "<0>Sei sicuro di voler eliminare <1></1>?</0><p>Non potrai tornare indietro</p>",
+        "title": "Cancella utente"
+      },
+      "password_reset": {
+        "description": "Cambia password per <1></1>",
+        "form": {
+          "label": "Nuova password",
+          "placeholder": "password",
+          "submit": "Cambia password"
+        },
+        "title": "Cambia password"
+      },
+      "table": {
+        "column_names": {
+          "action": "Azioni",
+          "admin": "Admin",
+          "photo_path": "Percorso foto",
+          "username": "Username"
+        },
+        "new_user": "Nuovo utente",
+        "row": {
+          "action": {
+            "change_password": "Cambia password",
+            "delete": "Elimina",
+            "edit": "Modifica",
+            "scan": "Scansiona"
+          }
+        }
+      },
+      "title": "Utenti"
+    }
+  },
+  "share_page": {
+    "media": {
+      "title": "Media condivisi"
+    },
+    "protected_share": {
+      "description": "Questa condivisione è protetta da una password.",
+      "title": "Condivisione protetta"
+    },
+    "share_not_found": "Condivisone non trovata",
+    "share_not_found_description": "Forse la condivisione è scaduta o è stata eliminata.",
+    "wrong_password": "Password sbagliata, prova di nuovo."
+  },
+  "sidebar": {
+    "album": {
+      "title": "Opzioni Album"
+    },
+    "download": {
+      "filesize": {
+        "byte": "{{count}} Byte",
+        "byte_plural": "{{count}} Bytes",
+        "giga_byte": "{{count}} GB",
+        "kilo_byte": "{{count}} KB",
+        "mega_byte": "{{count}} MB",
+        "tera_byte": "{{count}} TB"
+      },
+      "table_columns": {
+        "dimensions": "Dimensioni",
+        "file_size": "Peso",
+        "file_type": "Formato",
+        "name": "Nome"
+      },
+      "title": "Download"
+    },
+    "media": {
+      "exif": {
+        "exposure_program": {
+          "action_program": "Programma azione",
+          "aperture_priority": "Priorità apertura",
+          "bulb": "Bulb",
+          "creative_program": "Programma creativo",
```

Comment:
I'm not sure about this translation

Reply:
Creative program is a camera setting that has creative features like black and white

Reply:
Ok, it's correct then